docs/introduction/Sidechain-SDK-extension.rst

```diff
@@ -330,17 +330,16 @@ The Wallet by default keeps user secret info and related balances. The actual da
     void onRollback(byte[] version);
   }
 
-For example, a developer needs to have some event-based data, like an auction slot that belongs to him, and will start in 10 blocks and expire in 100 blocks. So in ApplicationWallet, he will additionally keep this event-based info and will react when a new block is going to be applied (onChangeBoxes method execution) to activate or deactivate that slot in ApplicationWallet.
-
+For example, a developer needs to have some event-based data, like an auction slot that belongs to him, that will start in 10 blocks and expire in 100 blocks. So in ApplicationWallet, he will additionally keep this event-based info and will react when a new block is going to be applied (onChangeBoxes method execution) to activate or deactivate that slot in ApplicationWallet.
```

Comment:
@zennermac 

For example, a developer needs to have event-based data, like an auction slot, that belongs to them and starts in 10 blocks and will expire in 100 blocks. The developer will keep this event-based info in the ApplicationWallet. THe developer will set the data to react by activating or deactivating a slot in ApplicationWallet when a new block is about to be applied (onChangeBoxes method execution).